frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -12,44 +13,87 @@ import { Theme } from "./themes";
 import type { ConfiguredRoute, Workflow, WorkflowConfiguration } from "./workflow";
 import ErrorBoundary from "./workflow";
 
+const FF_POLL = +process.env.REACT_APP_FF_POLL || 300000;
```

Comment:
What do you think about having a lower bound?

Reply:
I like the idea but I also think it's reasonable to not have one so that users can lower it if they feel like they want more frequent polling.